d8/d8batch/modules/d8feedsourcect/d8feedsourcect.info.yml

```diff
@@ -0,0 +1,10 @@
+name: D8: Feed Source CT
+type: module
+description: Contains the definition of the Feed Source CT.
+core: 8.x
+package: D7to8
+dependencies:
+  - field
+  - link
+  - node
+  - path
```

Comment:
Why depending on path.module? I thought these nodes have only a link field, but not a (custom) path.